java/configure_certs.sh

```diff
@@ -0,0 +1,19 @@
+# clone the test kmip server
+git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git
```

Comment:
I am concerned about linking to drivers-evergreen-tools.git in public documentation. The repository is intended for assisting driver testing.

The "best effort" note permits us making breaking changes. I still prefer to avoid changes in driver-evergreen-tools scripts breaking user examples or documentation.

Would it be OK to copy the needed scripts into this repository?

Reply:
Sounds good, agree it makes sense to not tie this tutorial to changes in the `drivers-evergreen-tools` repo. I've copied over the necessary KMIP scripts and certs into the `/.kmip` directory of this repo and updated them to work with the new structure.